src/gtk/toga_gtk/widgets/optioncontainer.py

```diff
@@ -21,3 +21,18 @@ def add_content(self, label, widget):
     def set_on_select(self, handler):
         # No special handling required
         pass
+
+    def remove_content(self, index):
+        return NotImplementedError
```

Comment:
This should be `self.interface.factory.not_implemented()`. Search the GTK backend for other examples of usage.